opesci/compilation.py

```diff
@@ -58,6 +58,15 @@ def __init__(self, cppargs=[], ldargs=[]):
         cppargs = ['-Wall', '-std=c++11', '-I%s/include' % get_package_dir()] + opt_flags + cppargs
         ldargs = ['-lopesci', '-Wl,-rpath,%s/lib' % get_package_dir(),
                   '-L%s/lib' % get_package_dir()] + ldargs
+
+        # Check if the g++ version is set in the GCCVERSION environment variable. If it is,
+        # then adds the sufix of the version to the g++, and puts it in the CC variable.
+        # i.e. GCCVERSION=4.8 then CC = g++-4.8
+        gccVersion = environ.get('GCC_VERSION')
+
+        if gccVersion:
+            environ['CC'] = 'g++-' + gccVersion
+
```

Comment:
I don't think overriding one environment variable based on the value of another "behind the scene" is great, since it is not obvious to the user what is happening. Can we not simply do `export CC="g++-4.8"` in our travis setup?